.github/workflows/testbench.yml

```diff
@@ -30,19 +30,21 @@ on:
 
 jobs:
   build-and-test:
-    runs-on: ubuntu-24.10
+    runs-on: ubuntu-24.04
 
     steps:
       - name: Checkout SOF repository (PR source)
         uses: actions/checkout@v4
         with:
           path: sof
 
+      # note: libasound-dev needed for testbench build which still
+      #       requires system ALSA headers to be present
```

Comment:
@singalsu @kv2019i testbench should use the same ALSA version as other tools to avoid any confusion.